src/vast_tracker.js

```diff
@@ -404,10 +404,10 @@ export class VASTTracker extends EventEmitter {
     const isAlwaysEmitEvent = this.emitAlwaysEvents.indexOf(eventName) > -1;
 
     if (trackingURLTemplates) {
-      this.emit(eventName, '');
```

Comment:
@eliamaino-fp Why was it an empty string here before ?

Reply:
Looks like legacy from the old 1.x API, this payload as empty string was not even documented in the public events API of the 2.x version of the client